ecosystem/typescript/sdk/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ All notable changes to the Aptos Node SDK will be captured in this file. This ch
 
 N/A
 
+## 1.3.16 (2022-10-12)
+
+- Add `estimatePrioritizedGasUnitPrice` to the simulation interface. If set to true, the estimated gas unit price is higher than the original estimate. Therefore, transactions have a higher chance to be executed during congestion period.
+
```

Comment:
Also add in, `estimateGasPrice` now provides a `prioritized` and a `deprioritized` gas unit price